### ISSUE TITLE
'High Contrast White' theme Tabs selected text color contrast

### ISSUE THREAD

**Issue by prsdthkr** [SAP/fundamental-styles#1038]
**Describe the bug**
The Tabs selected text color contrast ratio is less than 4.5:1
Also for selected/active tabs, fd-tabs__icon text color should be in line with the theme i.e white.

> What version of the Fundamental Library Styles are you using?
v0.10.0-rc.1

**To Reproduce**
1. Go to https://deploy-preview-1025--fundamental-styles.netlify.app/components/tabs.html
2. Check the Tabs selected text color contrast ratio

**Expected behavior**
The Tabs selected text color contrast ratio should be 4.5:1 or above. Ideally 7:1 or above for WCAG AAA compliance. #0049DB would be the best choice.

Text color for icon based tabs that are active should be white.

**Screenshots**
![image](https://user-images.githubusercontent.com/43764832/81863841-ab151880-9520-11ea-95f8-4ac38d376de5.png)
Fails all requirements: https://webaim.org/resources/contrastchecker/?fcolor=8FB5FF&bcolor=FFFFFF

![image](https://user-images.githubusercontent.com/43764832/81864886-16131f00-9522-11ea-8587-6fcc6125d6f3.png)

**Desktop (please complete the following information):**
 - OS: macOS
 - Browser: Chrome 
 - Version: 81.0.4044.129 (Official Build) (64-bit)

**Comment by Mike-Diaz** [comment]
Fixed as of version 0.13.0 - Contrast ratio is now 7.5:1

<img width="238" alt="Screen Shot 2020-10-08 at 5 53 39 PM" src="https://user-images.githubusercontent.com/21978807/95529218-3808b480-098f-11eb-9a95-0d2bad0c1767.png">

**Comment by prsdthkr** [comment]
![image](https://user-images.githubusercontent.com/43764832/96192563-1feae500-0efb-11eb-84f4-74cba722c896.png)
